jumanji/environments/packing/bin_pack/env.py

```diff
@@ -61,16 +70,16 @@ class BinPack(Environment[State]):
         - ems_mask: jax array (bool) of shape (obs_num_ems,)
             indicates the EMSs that are valid.
         - items: `Item` tree of jax arrays (float if `normalize_dimensions` else int32) each of
-            shape (max_num_items,),
+            shape (max_num_items, 6),
```

Comment:
Weren't these going to be flat?

Reply:
No, I only flatten them inside the actor_critic.py file.